packages/components/src/components/SquiggleChart.tsx

```diff
@@ -158,11 +164,6 @@ export const SquiggleChart: React.FC<SquiggleChartProps> = ({
   height = 60,
   width = NaN,
 }: SquiggleChartProps) => {
-  const target = React.useRef(null);
-  const [componentWidth] = useSize(target);
-  // I would have wanted to just use componentWidth, but this created infinite loops with SquiggleChart.stories.
-  //So you can manually add a width, as an escape hatch.
-  let _width = width || componentWidth;
```

Comment:
Why remove the useSize? I think this could bite us later, but I realize it's a bit hacky.

Reply:
didn't want the dead code once we were using a different solution, based on the `antd` factor-out